bin/diagnostics.bash

```diff
@@ -258,13 +293,35 @@ run_diagnostics_namespaced() {
     done
 }
 
+get_config_files() {
```

Comment:
Does it also work in a multi-wc scenario?

Reply:
This would only include config files found in `CK8S_CONFIG_PATH`, so it will only include the `wc` found in the current `CK8S_CONFIG_PATH`. To get config files for another `wc` you would have to change `CK8S_CONFIG_PATH`, atleast with how we do multi-wc atm

Reply:
Great, then I guess we only need to document it in public docs.

Reply:
Yes, I have a [PR prepared](https://github.com/elastisys/compliantkubernetes/pull/941) here and I will just wait until I merge this one.